src/components/Item/Quantity.js

```diff
@@ -1,28 +1,34 @@
 import React from 'react';
 import styled from 'styled-components';
 
-import { borderRadiusMedium, colorLightGrey, spacingLarge, borderWidthThick } from '../../styles/designTokens';
+import { borderRadiusMedium, colorLightGrey, borderWidthThick } from '../../styles/designTokens';
 import QuantityButton from './QuantityButton';
 import QuantityInput from './QuantityInput';
 
 const Wrapper = styled.div`
   grid-column-start: 1;
   grid-column-end: 3;
   display: grid;
+  grid-template-columns: 1fr 2fr 1fr;
   justify-items: center;
   border-radius: ${borderRadiusMedium};
-  grid-template-columns: ${spacingLarge} auto ${spacingLarge};
-  width: 40%;
+  width: 15rem;
   border: ${borderWidthThick} solid ${colorLightGrey};
-  height: 4.4rem;
 `;
 
-function Quantity() {
+function Quantity({ cartItemId, currentQuantity, handleIncrease, handleDecrease, handleUpdate }) {
+  const handleChange = function(value, itemId) {
+    const newValue = parseInt(value, 10);
+    handleUpdate(itemId, newValue);
+  };
+
   return (
     <Wrapper>
-      <QuantityButton disabled>-</QuantityButton>
-      <QuantityInput />
-      <QuantityButton>+</QuantityButton>
+      <QuantityButton onClick={() => handleDecrease(cartItemId)} disabled={currentQuantity < 2}>
```

Comment:
Don't create new function in JSX. This way JS needs to declare that method every time that component re-render.

Reply:
I need to call this function with parameters, how can I do that without this syntax?

Reply:
Just bind that `cartItemId` in container, and pass simple handler to that method. Also you can create handler for handleDecrease above `return(...)`, but this way you need to change the naming. I would bind everything in controller, it will simplify the code.